services/site/src/components/homepage/Card.tsx

```diff
@@ -48,7 +50,7 @@ const Card: React.FunctionComponent<CardProps> = ({
         </span>
       )}
 
-      {finishedLevels > 0 && finishedLevels !== levels && (
+      {((finishedLevels > 0 && finishedLevels !== levels) || (finishedLevels >= 0 && challengeStatus === ChallengeStatus.InProgress)) && (
```

Comment:
If I understand correctly we want to show progress indicator even if a user didn't complete the first level.
For that it would be enough to only check for `ChallengeStatus === ChallengeStatus.InProgress`, since the first part checks if the challenge has been started and the first level is completed and second  part basically only checks if the challenge status is in progress (`finishedLevels` will always be `>= 0`).